backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -32,6 +33,20 @@ def get_collection_details(collection_uuid: str, visibility: str, user: str):
         raise ForbiddenHTTPException()
 
 
+@db_session
+def create_collection(body: object, user: str = ""):
+    collection = Collection.create(
```

Comment:
remove the default value for user. If a user value is not passed in then, there must be something wrong with the token and we shouldn't proceed,